fw/compile-firmware.md

```diff
@@ -12,98 +12,76 @@ parent: Firmware
 
 # Introduction
 
-The [BastardKB QMK repository](https://github.com/bastardkb/bastardkb-qmk) contains the firmware code and releases for the Bastard Keyboards Charybdis boards.
+The [BastardKB QMK repository](https://github.com/bastardkb/bastardkb-qmk) contains the behavioural firmware code and releases for the Bastard Keyboards Charybdis boards.
 
-While that repository is used as primary source of truth for Bastard Keyboards firmwares as it contains the latest changes and improvements, the maintainers aim at upstreaming all those changes to the official [QMK repository](https://github.com/qmk/qmk_firmware).
+The keymaps are stored in the [BastardKB QMK userspace repository][bkbus].
 
-While the behavioral code is stored on the QMK repository, the keymaps are stored in our [QMK userspace](https://github.com/Bastardkb/qmk_userspace).
+The QMK repository repository is used as primary source of truth for Bastard Keyboards firmwares and contains the latest changes and improvements.
+The maintainers aim at upstreaming all those changes to the official [QMK repository](https://github.com/qmk/qmk_firmware).
 
-This page details how to compile your own firmware. Building from source is useful to people who want to customize their keyboard and keymaps beyond what Via offers. This involves using a command-line interface, writing C code, and compiling this code into the final firmware file.
+This page details how to build your own firmware. 
+Building from source is useful to people who want to customize their keyboard and keymaps beyond what Via offers. 
+You will have to modify the keymap `C` code, and from there compile your firmware either using Github actions or the local command line.
 
 If that seems too complicated, you can also use one of the default releases, or VIA.
```

Comment:
Doesn't one include the other? Perhaps "you can also use one of the defalult releases using via".

"default releases" should also link to the releases page.

Reply:
added release link. not sure what you mean with the other

Reply:
See suggestion below